src/makeRobot.js

```diff
@@ -39,6 +39,73 @@
  */
 function makeRobot(name, wheels, version) {
   // write code here
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, chip version: ${this.version}, wheels: ${
+        wheels}`;
+    },
+
+    get location() {
+      return `${this.name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goForward(value = 1) {
+      if (value < 1) {
+        return this;
+      }
+
+      this.coords.y += value;
+
+      return this;
```

Comment:
```suggestion
      if (value > 0) {
        this.coords.y += value;
      }

      return this;
```
do not repeat yourself)